multiplatform/clang_tidy_check/action.yml

```
@@ -0,0 +1,39 @@
+name: 'clang_tidy_check'
+description: 'Run clang tidy check.'
+
+inputs:
+
+  build_directory:
+    description: 'Directory where builds are'
```

```suggestion
    description: 'Binaries directory'
```

What do you think? If accepted, please apply to the platform actions as well.